modules/RTC/JitsiLocalTrack.js

```diff
@@ -340,7 +340,7 @@ JitsiLocalTrack.prototype._setMute = function (mute) {
             return self._sendMuteStatus(mute);
         })
         .then(function() {
-            self.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED);
+            self.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED, this);
```

Comment:
Because it makes implementation of such listeners harder and single listener instance can not be reused. I basically have to create separate listener instance for every track and store it somewhere in order to unregister it later. Like here for example I have one instance, but would have to create some map or whatever to store all of these:
https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/connectivity/ParticipantConnectionStatus.js#L236

Without passing the source object event listener interface feels incomplete.